src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/ImportOrganizationUserCommand.cs

```diff
@@ -196,14 +237,51 @@ private async Task<CommandResult<InviteOrganizationUsersResponse>> InviteUsersAs
         return await _inviteOrganizationUsersCommand.InviteImportedOrganizationUsersAsync(request);
```

Comment:
This isn't the vnext method, so it should be using `OrganizationService`.

I don't think you need to worry about the new command here at all - we'll roll it out across all callers when it's ready, for now this code can just rely on the existing `OrganizationService` method. The command is only being used for scim at the moment.